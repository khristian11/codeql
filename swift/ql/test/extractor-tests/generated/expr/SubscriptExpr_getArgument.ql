import codeql.swift.elements
import TestUtils

from SubscriptExpr x, int index
where toBeTested(x)
select x, index, x.getArgument(index)
