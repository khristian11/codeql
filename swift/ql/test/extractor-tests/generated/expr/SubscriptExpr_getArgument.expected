| expressions.swift:35:7:35:12 | ...[...] | 0 | : 1 |
| expressions.swift:88:1:88:6 | ...[...] | 0 | : a |
| expressions.swift:137:3:137:7 | ...[...] | 0 | : 1 |
| expressions.swift:138:10:138:17 | ...[...] | 0 | : 3 |
| expressions.swift:138:10:138:17 | ...[...] | 1 | : 4 |
